src/cudamatrix/cu-device.h

```diff
@@ -100,14 +103,15 @@ class CuDevice {
   /// will always be a multiple of n (from properties_.textureAlignment).
   /// Otherwise, return 16, which is the stride used for CPU matrices.
   int32 GetMatrixAlignment() const;
-  
+
  private:
   CuDevice();
   CuDevice(CuDevice&); // Disallow.
   CuDevice &operator=(CuDevice&);  // Disallow.
 
+
   static CuDevice global_device_;
-  
+  static cublasHandle_t handle_;
```

Comment:
This should probably not be a static variable-- in a situation in future where we had multiple CuDevice objects it would make sense for each to have its own handle.  You can make it a regular member and acccess it as global_device_.handle_.